apps/fipsinstall.c

```diff
@@ -197,6 +199,7 @@ typedef struct {
     unsigned int x942kdf_key_check : 1;
     unsigned int pbkdf2_lower_bound_check : 1;
     unsigned int ecdh_cofactor_check : 1;
+    unsigned int no_defer_tests : 1;
```

Comment:
We probably now need another option here that allows the self tests to run on demand now.
Since the 'defer_tests' no longer runs all tests, it may be nice for the user to be able to check that the tests will work when they do run.

Reply:
Uhmm I thought the change wa also causing the tests to be run immediately as I think fipsinstall uses its own configuration. If it doesn't I think I would still tie the two actions together, as I can't think of a case where it wuld make sense to want to force all tests at startup but not do it for fipsinstall itself.